File: JavaScript/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const PI = 3.14159
var raio = parseFloat(lines.shift());

volume = ((4 / 3.0) * (PI * Math.pow(raio, 3)))

console.log('VOLUME = ' + volume.toFixed(3))

File: Go/1011.go
package main

import (
	"fmt"
	"math"
)

const PI = 3.14159

func main() {

	var raio float64

	fmt.Scanf("%f", &raio)

	volume := ((4 / 3.0) * (PI * math.Pow(raio, 3)))

	fmt.Printf("VOLUME = %.3f\n", volume)
}
